lease/leasehttp/http.go

```diff
@@ -188,9 +188,14 @@ func TimeToLiveHTTP(ctx context.Context, id lease.LeaseID, keys bool, url string
 			errc <- err
 			return
 		}
-		b, err = ioutil.ReadAll(resp.Body)
-		resp.Body.Close()
+		b, err = readFunc(resp)
 		if err != nil {
+			// follower cannot reach leader at the moment
```

Comment:
The error message we got in the tester is `(rpc error: code = 2 desc = Post http://10.240.0.3:2380/leases/internal: EOF)`.   is `Post http://10.240.0.3:2380/leases/internal: EOF` return by http call equals to either io.EOF or io.ErrUnexpectedEOF?